packages/react/src/SelectPanel/SelectPanel.stories.tsx

```diff
@@ -51,8 +53,6 @@ export const Default = () => {
     <>
       <h1>Multi Select Panel</h1>
       <SelectPanel
-        title="Select labels"
```

Comment:
Remove all these because they are not default - let me know if you have any concern

Reply:
I think we do recommend titles for a11y, so I'd keep them in the examples

Reply:
That makes sense!